ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -3657,4 +3657,22 @@ def test104_edit_order_page_then_overview(self, edit_case):
         else:
             self.assertFalse(self.order_page.confirm_popup(check_only=True))
             self.info('asserting redirection to active table')
-            self.assertEqual(self.order_page.orders_url, self.base_selenium.get_url())
\ No newline at end of file
+            self.assertEqual(self.order_page.orders_url, self.base_selenium.get_url())
+
+    def test105_check_new_fields_are_displayed_in_XSLX(self):
```

Comment:
move this test case to extended file, you need to implement steps from: 6 , 7 "change lang to DE" you can make use of this test case
https://github.com/modeso-open-source/1lims-automation/blob/974dea4d920f014c5f0b8aeaee9e64ccfdc4531e/ui_testing/testcases/extended_tests/test001_order.py#L171